docs/test-cases/vcf-highlight/Conditions/Highlight_vcf_with_complex_conditions.md

```diff
@@ -0,0 +1,62 @@
+# Highlight variations depending on the selected profile with complex conditions
+
+Test verifies
+ - Background colour of Variations in the Variants panel and VCF track is highlighted according to the selected profile with complex conditions
+
+**Prerequisites**:
+
+1. dataset = **dm6_data**, .vcf = **[agnts3.09-28.trim.dm606.realign.vcf](https://ngb-oss-builds.s3.amazonaws.com/public/data/demo/ngb_demo_data/agnts3.09-28.trim.dm606.realign.vcf.gz)**
+2. **interest_profiles.json** that contains **'Complex Conditions'** profile with conditions
+```
+ "Complex Conditions" : {
+        "is_default" : true,
+        "conditions" : [
+            {
+                "highlight_color" : "0083ff",
+                "condition" : "(ReadPosRankSum > 1) and (ReadPosRankSum < 3) and (DP <= 200)"
+            },
+            {
+                "highlight_color" : "149150",
+                "condition" : "(DP in [386, 435]) and (SOR == 1.461)"
+            },
+            {
+                "highlight_color" : "ce632d",
+                "condition" : "(DP in [386, 435]) and ((SOR == 1.461) or (SOR == 1.227))"
+            },
+            {
+                "highlight_color" : "f92900",
+                "condition" : "(DP notin [597,587]) and (DP >= 543) and (MQ != 91.95)"
+            },
+              {
+                "highlight_color" : "ff00b2",
+                "condition" : "(FS > 412) or (QD > 36)"
+            }
+           ]
+         },
+
+
+```
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login to NGB as system admin user | |
+| 2 | Go to  **Settings** -> **VCF** tab | 'VCF' tab is displayed in the Settings' window|
+| 3 | Set checkbox in **Enabled** field (if it is not set)| **'Select the profile'** field is displayed |
+| 4 | Select **Complex Conditions** profile in dropdown list of **'Select the profile'** field|
+| 5 | Click **Save** button | <li> Changes successfully saved <li> 'Settings' window is closed | 
+| 6 | Go to **Datasets** tab | 
+| 7 | Select dataset and vcf file from **Prerequisites** | Vcf file (**agnts3.09-28.trim.dm606.realign.vcf**) is selected in dataset (**dm6_data**) 
+| 8 | Go to **Variants** panel | |
+| 9 | Find Variation on Position **chr X:12584271**| Background of variation highlight in **'0083ff'(blue)** color |
+| 10| Click on variation with Position **chr X:12584271** |Browser window is opened with vcf track|
+| 11| Look at variation on vcf track in Browser| Variation is highlighted in color  **'0083ff'(blue)** along all its length in expanded and collapsed view
```

Comment:
Add separate steps for switching between VCF-track view ("expanded" and "collapsed")